unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -119,24 +191,30 @@ export class CreatorLockForm extends React.Component {
             name="name"
             onChange={this.handleChange}
             defaultValue={name}
+            data-valid={valid.name}
           />
         </FormLockName>
         <FormLockDuration>
           <input
-            type="text"
+            type="number"
```

Comment:
This is good! We could start with a PR which does just that for example (and does not change any of the logic)